src/gc/gcinterface.h

```diff
@@ -41,6 +41,51 @@ typedef enum
     walk_for_loh = 3
 } walk_surv_type;
 
+// Arguments to GCToEEInterface::StompWriteBarrierResize and
+// GCToEEInterface::StompWriteBarrierInitialize. 
+struct WriteBarrierResizeArgs 
```

Comment:
Similar, it may be useful to have just one uber callback that takes flags about what all has changed. It should be more flexible than fine grained method because of it allows batching different updates together.